README.md

```diff
@@ -1,16 +1,64 @@
+<img height='175' src="https://user-images.githubusercontent.com/37406965/51083189-d5dc3a80-173b-11e9-8ca0-28015e0893ac.png" align="left" hspace="1" vspace="1">
+
+# Self Service Android Application for MifosX
```

Comment:
We have changed the name Self Service -> Mifos Mobile, So It should be  Mifos-Mobile Android Application for MifosX